tailwind.config.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     },
     // Renaming breakpoints temporary until we remove bootstrap usage
     screens: {
+      xsmall: "380px",
```

Comment:
Added a new breakpoint for mobile sizes to the tailwind config

Reply:
You should not need to make a new breakpoint for this situation

https://foundation.mozilla.org/en/style-guide/#breakpoints

In a mobile first approach to this there should be a reason to have different stylings between 380px and 576px

You should be able to think of it as 
1. Have classes that work all the way to 576px
2. then have classes for 576px-768px
etc